### DISCUSSION TITLE
Mic

### DISCUSSION THREAD

**Discussion by bilal201715** [speechly/speechly#316] [Q&A] [answered]
Hi, We notice that the mic is not working on ecommerce demo. Can you tell me when it will work and why it is not working?

**Comment by thisismatu** [comment]
Hi @bilal201715 and thanks for reporting this issue to us. Our tech team will take a look and ping you once the demo is working again. 👍

**Reply by bilal201715** [reply]
I can't wait to see it, Thanks

**Reply by thisismatu** [reply, marked as answer]
The issue has now been fixed and the demo should be working as normal. The underlying issue was a failure in model training, which caused the microphone button to appear unresponsive.
